src/VoiceConnection.ts

```diff
@@ -401,6 +401,7 @@ export class VoiceConnection extends EventEmitter {
 /**
  * Creates a new voice connection
  * @param joinConfig The data required to establish the voice connection
+ * @param options Options to create a connection
```

Comment:
The creation of a connection is already implied by the function's description on line 402.
```suggestion
 * @param options The connection options
```